.github/workflows/ci.yml

```diff
@@ -3,7 +3,10 @@
 
 name: Build and test CI
 
-on: [push]
+on:
+  push:
+  schedule:
+    - cron:  '0 20 17 1/1 *'
```

Comment:
If I look the next triggers:

```
next at 2021-08-17 20:00:00
then at 2021-09-17 20:00:00
then at 2021-10-17 20:00:00
then at 2021-11-17 20:00:00
then at 2021-12-17 20:00:00
```

What do you think to run at noon and midnight?

`0 */12 * * *`

Reply:
The staging server is stopped during the night. I will change the rules to every day from Monday to Friday at noon: 
`0 12 * * 1-5`

Reply:
Perfect.